snooty/rstparser.py

```diff
@@ -93,11 +92,10 @@ def parse_explicit_title(text: str) -> Tuple[str, Optional[str]]:
 def strip_parameters(target: str) -> str:
     """Remove trailing ALGOL-style parameters from a target name;
        e.g. foo(bar, baz) -> foo."""
-    match = PAT_PARAMETERS.search(target)
-    if not match:
+    if not target.endswith(")"):
         return target
 
-    starting_index = match.start()
+    starting_index = target.rfind("(")
```

Comment:
What are these changes? Not quite grasping how targets relate to tab titles.

Reply:
This looks like it's mistakenly reverting e4fa7fcd4b18126a3c38d0c1dc84cd882dbec909

Reply:
it is! which is strange since i did a `git pull upstream/master` and it says its up to date...im working on fixing it